src/connect.spec.lua

```diff
@@ -349,4 +349,36 @@ return function()
 		expect(ConnectedSomeComponent.SomeEnum).to.be.ok()
 		expect(ConnectedSomeComponent.SomeEnum.Value).to.equal(1)
 	end)
+
+	-- Issue https://github.com/Roblox/roact-rodux/issues/48
+	it("should never pass the store and innerProps to `mapStateToProps`", function()
+		local somePropValue = {}
+		local lastMappedProps = nil
+		local function mapStateToProps(state, props)
+			lastMappedProps = props
```

Comment:
I could do both, because initially I had the expect be only after the dispatch, but something else trigger another mapStateToProps call and overwrite the incorrect one.